lib/menu.rb

```diff
@@ -0,0 +1,18 @@
+class Menu
+    attr_reader :menu_items
```

Comment:
I can't see a test for this method. Please add one.

Reply:
Hi, How can I add a test for a attr_reader?